lib/dl_connector_clickhouse/dl_connector_clickhouse/formula/definitions/functions_math.py

```diff
@@ -138,7 +145,7 @@ class FuncLeastNumbersCH(base.FuncLeastBase):
     ),
     # exp
     base.FuncExp.for_dialect(D.CLICKHOUSE),
-    # fdiv_safe
+    # div_safe
```

Comment:
Well this was unnecessary
`FDIV_SAFE` is the float version of the `DIV_SAFE` func, bring this comment back please